temoa/components/operations.py

```
@@ -228,22 +194,104 @@ def create_operational_vintage_sets(model: TemoaModel) -> None:
         model.is_seasonal_storage[t] = t in model.tech_seasonal_storage
 
 
-def ramp_up_day_constraint(
+def _ramp_activity_increase(
     model: TemoaModel,
+    ramp_up: bool,
```

Ruff (0.16.0)

[warning] Boolean-typed positional argument in function definition

(FBT001)